content/mobile.md

```diff
@@ -452,6 +452,33 @@ What is often confusing to people is that setting `App.accessRule` is not enough
 
 To get around these restrictions, you'll have to use what is known as [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS). In contrast to the whitelisting mechanism configured on the client, CORS relies on headers set by the server. In other words, in order to allow access to a remote resource, you may have to make configuration changes on the server, such as setting a `Access-Control-Allow-Origin` header.
 
+<h3 id="csp">Runtime Permissions</h3>
```

Comment:
I might say `System Permission` since that is what Google calls it?

https://developer.android.com/training/permissions/requesting.html